src/balena.sbvr

```diff
@@ -649,6 +663,16 @@ Fact type: release has release type
 	Necessity: each release has exactly one release type.
 	Definition: "final" or "draft"
 
+Fact type: release has peak memory usage
+	Necessity: each release has at most one peak memory usage
+Fact type: release has peak cpu usage
+	Necessity: each release has at most one peak cpu usage
+Fact type: release has average memory usage
+	Necessity: each release has at most one average memory usage
+Fact type: release has average cpu usage
+	Necessity: each release has at most one average cpu usage
+Fact type: release has run time
```

Comment:
Not sure whether we would care to have a rule to only allow this being set iff it's not still running, but we can probably add that later if we need.